### ISSUE TITLE
[Bug]: Deleting tech nodes was lost at some point?

### ISSUE THREAD

**Issue by TsunamiAnt** [BeeStation/BeeStation-Hornet#11857]
### BYOND Version

514.1589

### Round Date

2024-11-08

### Round ID

50592

### Test merges

[#9124: 'Enable serverside 515.1614' by Crossedfall at commit 1b8f225789](https://www.github.com/beestation/beestation-hornet/pull/9124)
[#11827: 'Nanite active program conversion, rebalancing & additions' by PowerfulBacon at commit 6ba8b1eb71](https://www.github.com/beestation/beestation-hornet/pull/11827)

### Describe the bug

You used to be able to use the science server control console to delete researched tech nodes. I think we might have lost that during the overheating update?

Really annoying.

### What did you expect?

I expected to be able to delete science nodes.

### Reproducible steps

log into the science server control console.
see that you cannot delete techs now.

**Comment by PowerfulBacon** [comment]
I've always heard about deleting research from when I started playing, but I've never actually seen the ability to delete research myself. When I tried years ago it wasn't possible

**Comment by MarkusLarsson421** [comment]
Considering research is a very... lame mechanic and is mostly an annoying obstacle. Good this doesn't work. Being able to simply delete research would be really annoying, especially if the Scientists aren't particularly good at it.

**Comment by TsunamiAnt** [comment]
Says the scientist. If the ai wants to stop sec from printion ions it'd be a million times better to cut it off at the tech level than unlinking their lathe or some shit pike that.

**Comment by MarkusLarsson421** [comment]
> Says the scientist.

No, says the non-Science related personnel. Whenever I play R&D I more or less speedrun it as it is so stale. If I encounter problems I adapt. However, not everyone is a pro-gamer and of course not. So quite often does Science have the issue of being almost hard-stuck on tier 1 and never progressing, which can be quite annoying for other departments waiting and or relying on Science. Say if I'm in Engineering wanting to build a shuttle, but I can't finish it because we don't have the tech nor do we have the resources because the ORM is still tier 1 too. Point being, we shouldn't make it even more difficult for those who DON'T know R&D.

> If the ai wants to stop sec from printion ions it'd be a million times better to cut it off at the tech level than unlinking their lathe or some shit pike that.

Yeah, they could unlink it, disable power, blow it up, or just take the boards themselves! EZ
Would take an Engineering Cyborg 2 seconds to nick both of them and destroy them in disposals or something.
